### PR TITLE
fix services v2 python snippets

### PR DESCRIPTION
**Pull request by kpenfound**
Tested:

docs/current/guides/snippets/use-services/expose-host-services-to-container/index.ts
docs/current/guides/snippets/use-services/expose-host-services-to-container/main.py

docs/current/guides/snippets/use-services/expose-service-containers-to-host/index.ts
docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py

docs/current/guides/snippets/use-services/start-stop-services/index.ts
docs/current/guides/snippets/use-services/start-stop-services/main.py

### CODE REVIEW

File: docs/current/guides/snippets/use-services/expose-host-services-to-container/main.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ async def main():
     # create Dagger client
     async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
         # expose host service on port 3306
-        host_srv = client.host().service(frontend=3306, backend=3306)
+        host_srv = client.host().service([dagger.PortForward(frontend=3306, backend=3306, protocol='TCP')])
```
Comment (helderco) [inline docs/current/guides/snippets/use-services/expose-host-services-to-container/main.py:12]: Should use the enum for protocol and (nit) prefer to follow the defined [parameter order](https://dagger-io.readthedocs.io/en/latest/client.html#dagger.PortForward):
```suggestion
        host_srv = client.host().service([dagger.PortForward(backend=3306, frontend=3306, protocol=dagger.NetworkProtocol.TCP)])
```

File: docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py [hunk 2]
```diff
@@ -23,13 +25,15 @@ async def main():
...
-        tunnel = client.host().tunnel(http_srv).start()
+        tunnel = await client.host().tunnel(http_srv).start()
 
         # get HTTP service address
-        tunnel.endpoint()
+        endpoint = await tunnel.endpoint()
 
         # access HTTP service from host
-        # commenting below as it's a blocking function, needs changes
+        r = requests.get("http://" + endpoint)
```
Comment (helderco) [inline docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py:34]: requests is sync/blocking. [Use httpx](https://www.python-httpx.org/async/) instead:
```suggestion
        async with httpx.AsyncClient() as http:
            r = await http.get(f"https://{endpoint}")
```

**Review by vito — APPROVED** [review]

File: docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py (hunk 2 above)
Comment (kpenfound) [reply docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py]: Updated!

File: docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 import dagger
 
+import httpx
```
Comment (helderco) [inline docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py:7]: These imports are not sorted, so you'll get a linting failure. It's builtins, then third party than own, so:

```
import sys

import anyio
import dagger
import httpx
```

Depending on how the sdk is installed, dagger could go on its own line in the bottom. What matters is ci, just run `./hack/make sdk:python:lint`.

To re-format locally, having `hatch` installed, it's just `hatch run lint`.

Comment (kpenfound) [reply docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py:7]: I can't get `hatch run lint` to work `line 1: lint: command not found`

And linting is failing for both:
```python
import sys

import anyio
import dagger
import httpx
```

and


```python
import sys

import anyio
import httpx

import dagger
```

Any idea?

Comment (helderco) [reply docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py:7]: Hatch needs to be run from `sdk/python`.

Comment (helderco) [reply docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py]: Pushed

**Review by helderco — APPROVED** [review]
Just the DCO now 👍

**Pull request merged**